### PR TITLE
Elekid WLoopCAN library port

### PR DESCRIPTION
**Pull request by Nushaab-Syed**
- modified makefile
-  modified tx implementation, getting rid of the old version and replacing it with WLoopCAN version

Some notes:
- Sensing and logic was not modified
- Only 5V and 24V (arb ID 30 and 32) were used, while 12V (arb ID 31) was not implemented prior
- ISense[] is passed in, while VSense[] is not as only 4 bytes are allocated for current only in each elekid message

### CODE REVIEW

**Comment by ren-andy** [comment]
lgtm, but I think @GuyBlumenthal should give the approval

File: STM32F303K6Tx/Core/Src/main.c [hunk 1]
```diff
@@ -23,6 +23,7 @@
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
 #include "string.h"
+#include "can.h"
```
Comment (GuyBlumenthal) [inline STM32F303K6Tx/Core/Src/main.c]: You shouldn't need this but whatevs

File: STM32F303K6Tx/Core/Src/main.c [hunk 2]
```diff
@@ -60,10 +62,6 @@ uint8_t V_byte[4];
 uint8_t I_byte[4];
 float offset[4] = {0.0, 0.0, 0.0, 0.0};
 uint8_t IDs[2] = {0x30, 0x32};
```
Comment (GuyBlumenthal) [inline STM32F303K6Tx/Core/Src/main.c]: I don't think you need these. You should probably use the IDs defined the config.h file in the can lib

File: STM32F303K6Tx/Core/Src/main.c [hunk 3]
```diff
@@ -75,6 +73,7 @@ static void MX_DMA_Init(void);
 static void MX_ADC2_Init(void);
 static void MX_CAN_Init(void);
 static void MX_TIM2_Init(void);
+static void MX_TIM16_Init(void);
 /* USER CODE BEGIN PFP */
 void float2Bytes(float val, uint8_t *bytes_array);
```
Comment (GuyBlumenthal) [inline STM32F303K6Tx/Core/Src/main.c]: You don't need this function anymore

**Review by GuyBlumenthal — COMMENTED** [review]
Looks good overall, just a few comments

**Review by GuyBlumenthal — APPROVED** [review]
lgtm

**Pull request merged**